clients/da-vinci-client/src/main/java/com/linkedin/davinci/stats/VeniceVersionedStatsReporter.java

```diff
@@ -31,21 +27,8 @@ public VeniceVersionedStatsReporter(
 
     this.isSystemStore = VeniceSystemStoreUtils.isSystemStore(storeName);
 
-    /**
-     * All stats which are aware of version should line up with Helix listener. It's duplicate
```

Comment:
what's the reason to remove this?

Reply:
Its replaced by the new metric. Previously it was using some shady `static final AtomicBoolean` for no reason.